planning/route_handler/src/route_handler.cpp

```diff
@@ -718,9 +718,15 @@ bool RouteHandler::getNextLaneletWithinRoute(
   if (exists(goal_lanelets_, lanelet)) {
     return false;
   }
+  lanelet::ConstLanelet start_lanelet;
+  bool flag_check = true;
+  if (!getClosestLaneletWithinRoute(route_msg_.start_pose, &start_lanelet)) {
+    flag_check = false;
+  }
```

Comment:
```suggestion
    const auto flag_check = getClosestLaneletWithinRoute(route_msg_.start_pose, &start_lanelet);
```
some refactoring.

Reply:
Also, I've check your code, and it doesn't seems to be able to be build.
It says `route_msg_` was not declared.

Reply:
The reason for the error is not the latest code. I have updated the code to the newest version.Can you review it